lib/ex_doc/formatter/html/autolink.ex

```diff
@@ -179,6 +181,13 @@ defmodule ExDoc.Formatter.HTML.Autolink do
   end
   defp strip_parens(string, _), do: string
 
+  defp split_string_to_link(string) do
+    case :binary.match(string, "(") do
```

Comment:
0c851c6 here it is.
I thought it should be OK here to exact-match against 2- or 1-element lists, correct?